bin/ch/ch.cpp

```diff
@@ -5,6 +5,7 @@
 #include "stdafx.h"
 #include "Core/AtomLockGuids.h"
 #include <CommonPal.h>
+#include <regex>
```

Comment:
Can we not add dependency to regex and fix the line ending instead?

Reply:
One problem it might cause is that the character offset for line/col numbers in the generated byte code will be different then JS file on disk

Reply:
I believe @tcare 's investigation into line endings with gitattributes suggests that the strongest defense we have is core.autocrlf, which we can't enforce. When we regen bytecode, could we also re-write the file we are generating bytecode after normalization back to disk?

Reply:
I don't think modifying the source JS files is a great idea, since this process isn't just used for code that we own, it's also used on WWAs (and in particular I think on WWAs that are not on the developer's machine, but on the end-user's machine)

Reply:
This is only intended to be for internal library js files.  WWA shouldn't hit this path?

Reply:
> One problem it might cause is that the character offset for line/col numbers in the generated byte code will be different then JS file on disk

This is exactly the problem this is trying to solve. Since we can't reliably enforce line endings on the javascript files from source control. It is simpler to normalize the line endings so we get the same line offset wherever the header is generated.

> When we regen bytecode, could we also re-write the file we are generating bytecode after normalization back to disk?

The line endings of the file is not actually wrong, it just depends on the setting on one's computer.

> I don't think modifying the source JS files is a great idea, since this process isn't just used for code that we own, it's also used on WWAs (and in particular I think on WWAs that are not on the developer's machine, but on the end-user's machine)

This change is localized to our testing host, therefore, it doesn't affect WWA.